### PR TITLE
Feature/chart improvements

### PR DESCRIPTION
**Pull request by hjoaquim**
- Tries to make the `ChartIndicators` more visible and usable.
- Experiment with the signatures of charting extension `to_chart()` and `OBBject.to_chart()`
   - idea was to wrap `OBBject.to_chart()` with `to_chart()` signature so that user can leverage the `kwargs` - although it works and make the updated signature available, I'm not sure if it enhances UX.

This is a different approach than the one proposed [here](https://github.com/OpenBB-finance/OpenBBTerminal/pull/5938)  with the goal of enhancing UX.

### CODE REVIEW

File: openbb_platform/extensions/charting/README.md [hunk 1]
```diff
@@ -90,4 +90,35 @@ The user can use any number of `**kwargs` that will be passed to the `PlotlyTA`
...
+    )
+    res.to_chart(**{"indicators": indicators})
+
+  ```
+
+- Get all the available indicators
+
+    ```python
+
+    from openbb_charting.core.plotly_ta.data_classes import ChartIndicators
```
Comment (deeleeramone) [inline openbb_platform/extensions/charting/README.md:121]: What do you think about a Charti object method within the extension framework of your other PR, `data.chart.available_indicators()`?

Comment (hjoaquim) [reply openbb_platform/extensions/charting/README.md:121]: Can easily be implemented as well! I mean, everything in this PR can easily be translated into the other!

Comment (deeleeramone) [reply openbb_platform/extensions/charting/README.md:121]: It might better to reflect on all chart improvements together.

Comment (hjoaquim) [reply openbb_platform/extensions/charting/README.md:121]: Haven't group them bc not sure if the OBBject extension should be the way moving forward. 😬

**Comment by hjoaquim** [comment]
Closing in detriment of https://github.com/OpenBB-finance/OpenBBTerminal/pull/5938.

**Pull request closed without merging**